Signal/src/ViewControllers/MessageMetadataViewController.swift

```diff
@@ -280,13 +280,24 @@ class MessageMetadataViewController: OWSViewController {
                 bodyLabel.numberOfLines = 10
                 bodyLabel.lineBreakMode = .byWordWrapping
 
-                let bubbleView = UIView()
-                bubbleView.backgroundColor = isIncoming ? UIColor.jsq_messageBubbleLightGray() : UIColor.ows_materialBlue()
+                let bubbleImageData = isIncoming ? ConversationViewController.incomingBubbleImageData() : ConversationViewController.outgoingBubbleImageData()
+                let leadingMargin: CGFloat = isIncoming ? 15 : 10
+                let trailingMargin: CGFloat = isIncoming ? 10 : 15
```

Comment:
Will this be right for RTL?

Reply:
Good catch. It was in line with what we were doing in conversation view, but our bubbles in conversation view were already weird for RTL. I've addressed that in 
https://github.com/WhisperSystems/Signal-iOS/pull/2586/commits/d202b7ce3fae3b9089868bd3304425285fdbdd2e

PTAL @charlesmchen

Reply:
<img width="432" alt="screen shot 2017-09-28 at 3 24 14 pm" src="https://user-images.githubusercontent.com/217057/30986605-04600150-a462-11e7-8b68-ae152225c25d.png">
<img width="432" alt="screen shot 2017-09-28 at 3 24 08 pm" src="https://user-images.githubusercontent.com/217057/30986604-045df82e-a462-11e7-86ae-b286db9c57bc.png">

Reply:
Actually, I'm going to merge this as it was, with your CR, but then open a new pr for the RTL bubble fix.